plugin/manager_linux.go

```diff
@@ -61,7 +61,7 @@ func (pm *Manager) enable(p *v2.Plugin, c *controller, force bool) error {
 	if err := pm.executor.Create(p.GetID(), *spec, stdout, stderr); err != nil {
 		if p.PluginObj.Config.PropagatedMount != "" {
 			if err := mount.Unmount(propRoot); err != nil {
-				logrus.Warnf("Could not unmount %s: %v", propRoot, err)
+				logrus.Warnf("Failed to %v", err)
```

Comment:
~Should this be "Failed to unmount" ?~ Ah, I see; the `.Error()` returns the "op" as part of the error, so this will include `unmount`.

Perhaps we should prefix that error with `"failed to <op>"` otherwise we'll have to add that everywhere 🤔

Reply:
I am following the style of errors from `os` package (and the general unix error style) which is in the form `<op>: <error>`. In fact it makes sense here as well, something that would look like:

```
... level=warning msg="umount <dir>: device busy"
```

which gives no less info than
```
... level=warning msg="Failed to umount <dir>: device busy"
```

so I'll drop the "failed to" from this place.